docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -35,6 +35,81 @@ The system repeatedly shows a warning message about overriding an inherited asse
 
 This warning, appropriate for the first assessment, incorrectly reappears on subsequent reassessments, suggesting that the application is still inheriting its assessment, even after it has been overridden. link:https://issues.redhat.com/browse/MTA-1825[MTA-1825]
 
+.Custom rule for javax.* package import is not triggered when uploaded as an XML file
+
```

Comment:
This is a specific example taken from our internal test cases. I think the text here can be generelized. The root cause of this issue is that - the Java search patterns that use IMPORT or PACKAGE as location, should not end in `.{*}` but just `{*}`. In order to solve this, whenever a custom rule has a pattern ending in `.{*}` can be changed to `{*}`. We can then use this specific example to explain the solution. Notice   `<javaclass references="javax.{*}">` in the example XML rule. If the users fix that to `<javaclass references="javax{*}">`, the rule will be converted correctly.

Reply:
@anarnold97 Not sure about this sentence:
```The root cause of this issue is - Java search patterns that use end in .{*} must use PACKAGE as location, not IMPORT```

In the reality:
 The root cause of this issue is - Java search patterns that use end in **{*}** must use PACKAGE as location, not IMPORT